bitcoin/src/address.rs

```diff
@@ -49,7 +49,7 @@ use crate::blockdata::script::{
 use crate::crypto::key::{PublicKey, TapTweak, TweakedPublicKey, UntweakedPublicKey};
 use crate::error::ParseIntError;
 use crate::hash_types::{PubkeyHash, ScriptHash};
-use crate::network::constants::Network;
+use crate::p2p::network::Network;
```

Comment:
Reexporting `Network` in `p2p` would look better IMO.

Reply:
Yeah a re-export `p2p::Network` exists in this PR, I use the real location in our import statements though. I've got in the habit because of not using re-exports in `lib.rs` but I guess this one is different and we could use the re-export?

Reply:
If there's a reexport I'd use it everywhere. But not a big deal, don't change the whole PR just for this.

Reply:
FTR this is resolved.